### ISSUE TITLE
"Jump without fighters?" confirmation dialog

### ISSUE THREAD

**Issue by DingusShingleton** [endless-sky/endless-sky#1685]
Sometimes in haste I hit J without realizing that my fighters are still deployed. Then I have to either waste time and fuel going back to get them or just save scum; not a great choice. A simple pop up that says 

"Jump without fighters?" 
[**J**ump] [**D**ismiss]

 would solve it. If you actually want to jump, your finger is already on J, so you lose barely a split second once you understand how it works (which is as soon as you do it the first time, as it's pretty intuitive).

**Comment by Amazinite** [comment]
This should be a preference, because right now the information to the player that they're going to leave ships behind is in the bottom left when ship outlines turn red. This would just be a more obvious way of expressing that to the player. Maybe a different alarm sound for when you're going to leave ships behind?

**Comment by DingusShingleton** [comment]
Problem is, the icons turn red only in the time between pressing J and jumping. If your jump speed is fast enough or you are moving slowly, you jump before you have a chance to react. 

Having said that, it would be fine as a preference (which I would immediately turn on and never turn off lol)

**Comment by jafdy** [comment]
Use **J**ump and **C**ancel, dismiss sounds to me like it would carry on and jump any way.

**Comment by DingusShingleton** [comment]
I used D because it is the standard hotkey for going back, be it [Dismiss] or [Done].

Better still, you could also use {**J**ump] and [**D**on't Jump], thus maintaining the **D** hotkey and making the choice completely explicit.

**Comment by Admiral-Danforth** [comment]
I vote for this

**Comment by Elyssaen** [comment]
I like this. Especially how you've thought through double tapping **J** and **D** so well.

**Comment by petervdmeer** [comment]
A pop-up effectively pauses the game, and I don't like a pause to appear while starting a jump. Pausing here would take the "flow/speed" out of the game.

If we would implement an option to support this, then I would prefer to handle this through the fleet jump command in #4124. (That fleet jumps also include recalling of fighters.)

**Comment by Amazinite** [comment]
I agree with @petervdmeer. Using the fleet jump command (shift+J) or perhaps also holding the jump key should recall your fighters. The fleet jump command should wait until fighters are recalled before jumping, while just using the jump key should jump regardless of whether or not your fighters are docked when you release the key.
